Formula/ncsdk.rb

```diff
@@ -0,0 +1,42 @@
+class Ncsdk < Formula
+  desc "Software Development Kit for the Neural Compute Stick"
+  homepage "https://github.com/movidius/ncsdk"
+  url "https://github.com/movidius/ncsdk/archive/v1.12.00.01.tar.gz"
+  sha256 "aceca6ffa87c87c2b29f5a89ed97a96f0b0238ff16d9c45e61e3e891a63c6386"
+
+  option "with-test", "Run build-time check"
+  option "without-python", "Build without python support"
+
+  depends_on "libusb"
+  depends_on "python3" => :recommended
+
+  patch do
```

Comment:
`New formulae should not require patches to build. Patches should be submitted and accepted upstream first.`